.github/workflows/tests.yml

```diff
@@ -59,6 +60,11 @@ jobs:
         mypy rest_api --exclude=rest_api/build/ --exclude=rest_api/test/
         echo "=== ui/ ==="
         mypy ui --exclude=ui/build/ --exclude=ui/test/
+    - uses: act10ns/slack@v1
+      with:
+        status: ${{ job.status }}
+        channel: '#haystack'
+      if: failure()
```

Comment:
We should send a message only if the failure happened on `master`, so I think we should do `if: failure() && github.ref == 'refs/heads/master'`